packages/plugins/rum/src/sourcemaps/sender.ts

```diff
@@ -0,0 +1,182 @@
+// Unless explicitly stated otherwise all files in this repository are licensed under the MIT License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2019-Present Datadog, Inc.
+
+import type { Logger } from '@dd/core/log';
+import type { GlobalContext } from '@dd/core/types';
+import retry from 'async-retry';
+import FormData from 'form-data';
+import fs from 'fs';
+import PQueue from 'p-queue';
+import type { Gzip } from 'zlib';
+import { createGzip } from 'zlib';
+
+import type { RumSourcemapsOptionsWithDefaults, Sourcemap } from '../types';
+
+import type { Metadata, Payload } from './payload';
+import { getPayload } from './payload';
+
+const errorCodesNoRetry = [400, 403, 413];
+const nbRetries = 5;
+
+export const getApiUrl = (context: GlobalContext) => {
+    const siteUrl = context.auth?.endPoint || 'datadoghq.com';
+    return `https://sourcemap-intake.${siteUrl}/api/v2/srcmap`;
```

Comment:
Same as prior, followed `datadog-ci`'s implementation, where it's not configurable (yet).

https://github.com/DataDog/datadog-ci/blob/026557808bb4ca0756bac078b715f63c9c11bca7/src/helpers/base-intake-url.ts#L1-L9

I do see where it could be valuable to make it configurable, but I prefer to cross this bridge when we get there. 

For the sake of simplicity.

Reply:
🤔 this may be actually needed before we release this plugin after all, given that `datadog-ci`'s offers the `DATADOG_SOURCEMAP_INTAKE_URL`.

Reply:
Added it in the last commit https://github.com/DataDog/build-plugins/pull/78/commits/19e38629f4d57a9508ed92def24b93a28336c3e1